modules/modifying-ovs-log-level-as-one-time-scenario.adoc

```diff
@@ -0,0 +1,80 @@
+[id="modifying-ovs-log-level-one-time-scenario"]
+= Modifying the OVS log level as a one-time scenario
+
+To modify the OVS log level in a one-time scenario, follow this procedure.
+
+.Procedure
+
+. Connect to the node in debug mode:
++
+[source,terminal]
+----
+$ oc debug node/<node>
+----
++
+[source,terminal]
+----
+$ chroot /host
+----
+
+. After access is established, check current syslog level for OVS modules:
++
+[source,terminal]
+----
+$ ovs-appctl vlog/list
+                 console    syslog    file
+                 -------    ------    ------
+backtrace          OFF       INFO       INFO
+bfd                OFF       INFO       INFO
+bond               OFF       INFO       INFO
+bridge             OFF       INFO       INFO
+bundle             OFF       INFO       INFO
+bundles            OFF       INFO       INFO
+cfm                OFF       INFO       INFO
+collectors         OFF       INFO       INFO
+command_line       OFF       INFO       INFO
+connmgr            OFF       INFO       INFO
+conntrack          OFF       INFO       INFO
+conntrack_tp       OFF       INFO       INFO
+coverage           OFF       INFO       INFO
+ct_dpif            OFF       INFO       INFO
+daemon             OFF       INFO       INFO
+daemon_unix        OFF       INFO       INFO
+dns_resolve        OFF       INFO       INFO
+dpdk               OFF       INFO       INFO
+...
+----
++
+
+Define the new verbosity required in the `/etc/systemd/system/ovs-vswitchd.service.d/10-ovs-vswitchd-restart.conf` file. In this example, the verbosity is changed from `info` to `dbg`:
++
+[source,text]
+----
+Restart=always
+ExecStartPre=-/bin/sh -c '/usr/bin/chown -R :$${OVS_USER_ID##*:} /var/lib/openvswitch'
+ExecStartPre=-/bin/sh -c '/usr/bin/chown -R :$${OVS_USER_ID##*:} /etc/openvswitch'
+ExecStartPre=-/bin/sh -c '/usr/bin/chown -R :$${OVS_USER_ID##*:} /run/openvswitch'
+ExecStartPost=-/usr/bin/ovs-appctl vlog/set syslog:dbg
+ExecReload=-/usr/bin/ovs-appctl vlog/set syslog:dbg
+----
++
+Change the last two strings by setting `syslog:<loglevel>` to `off`, `emer`, `err`, `warn`, `info`, or `dbg`. `off` filters out all messages.
+
+. Restart the service:
++
+[source,terminal]
+----
+$ systemctl daemon-reload
+----
++
+[source,terminal]
+----
+$ systemctl restart ovs-vswitchd
+----
+
+. Gather the logs, then edit the OVS log level to revert to the former value to prevent issues, such as this error:
```

Comment:
This step indicates to repeat the procedure and set the log level to the previous value, right?

+100 for including the log example if the value isn't reverted to the original value.

Reply:
Yes, because if you change the file manually(without applying MachineConfig), machine config operator will be confused about why the files it manages are changed.

But we may add a reminder at the end of the document to change log level back for MachineConfig option also (just to not have too much log messages)